src/common-ui/components/IndexDropdown.js

```diff
@@ -74,10 +78,57 @@ class IndexDropdown extends PureComponent {
         return this.placeholder.toLowerCase()
     }
 
+    renderTags() {
+        const tagOptions = this.props.tags.map((tag, i) => (
+            <span className={styles.tagPill}>
+                {tag}
+                {
+                    <button
+                        onClick={this.props.handleTagSelection(i)}
+                        className={styles.cross}
+                    />
+                }
+            </span>
+        ))
+        return tagOptions
+    }
+
+    renderDomains() {
+        const domainOptions = this.props.tags.map((tag, i) => (
+            <span className={styles.domainPill}>
+                {tag}
+                {
+                    <button
+                        onClick={() => this.props.handleTagSelection(tag)}
+                        className={styles.cross1}
+                    />
+                }
+            </span>
+        ))
+        return domainOptions
+    }
+
     render() {
+        // console.log(this.placeholder)
         return (
             <div className={this.mainClass} ref={this.props.setTagDivRef}>
+                {!this.props.isForSidebar ? (
```

Comment:
Let's change this to

```suggestion
                {!this.props.isForSidebar && (
```